### DISCUSSION TITLE
My account has been flagged by someone or somehow. Please open it for me, Github is really important to me. I have contributed a lot to the community.

### DISCUSSION THREAD

**Discussion by hoangchungk53qx1** [community/community#75521] [New to GitHub]
My account has been flagged by someone or somehow. Please open it for me. I have contributed a lot to the community.

@Akash1134 please help me

**Comment by hoangchungk53qx1** [comment]
Please Help me

**Comment by hoangchungk53qx1** [comment]
My ticket #2434591

**Comment by ghostinhershell** [comment]
Hi @hoangchungk53qx1! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
